### ISSUE TITLE
[Unstable] Acid Grenade quality does nothing

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#10350]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The acid grenade doesn't have an explosion/damage property of its own as it spawns an item.
Increasing quality therefore does not increase damage/radius.

Add an initial explosion back to the grenade, increase quality modifiers.
A small initial stun helps anyway because you can sprint away from the acid very easily

```xml
    <ItemComponent characterusable="false">
      <!-- statuseffect that explodes the grenade when used by something else than a character (e.g. a detonator) -->
      <StatusEffect type="OnUse" target="This" Condition="-100" />
      <StatusEffect type="OnBroken" target="This">
        <sound file="Content/Items/Weapons/StunGrenade.ogg" />
        <Explosion range="500" ballastfloradamage="10" itemdamage="20" force="0.5" sparks="true" shockwave="false" flames="false" smoke="false" flash="true">
          <Affliction identifier="acidburn" strength="25" />
          <Affliction identifier="stun" strength="0.5" />
        </Explosion>
      </StatusEffect>
    </ItemComponent>
    <Quality>
      <QualityStat stattype="ExplosionRadius" value="0.1" />
      <QualityStat stattype="ExplosionDamage" value="0.1" />
    </Quality>
```

### Version

0.20.5.0 (Unstable)

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@acf8740.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
